app/components/search/SearchResults/SearchResults.tsx

```diff
@@ -64,14 +50,10 @@ const SearchResults = ({ searchResults, expandList, setExpandList }) => {
           No results found in your area. Try a different location, category, or
           search term.
         </div>
-        {hits.map((hit, index) => (
-          <SearchResult
-            hit={hit}
-            index={index}
-            setCenterCoords={setCenterCoords}
-            key={hit.id}
-          />
-        ))}
+        {hits.length &&
```

Comment:
What's the purpose of this check? Won't `hits.map( ... )` have the same behavior regardless of whether there are 0 or more hits? Actually, I think what you have here will differ only in that it'll render a literal `0` onto the page if there are zero hits, since `{foo && bar}` will return `0` if `foo` is `0`, and if you return a 0, React will render that 0, similar to any other number. The `&&` operator in a React conditional rendering expression is only safe to use if the left side of it is a boolean, `null`, or `undefined`. If it's a number, then although `0` is treated as false in boolean expression, it will still return `0`, the number, which will be rendered literally onto the page.

Reply:
Oh that's a weird gotcha about the 0. I think I added this in response to some error from the compiler, which is no longer a problem after updating some of the types